packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -23,21 +23,45 @@ use axum::{
 use error_stack::Report;
 use futures::TryFutureExt;
 use include_dir::{include_dir, Dir};
+use serde::Deserialize;
 use utoipa::{
     openapi::{self, schema, schema::RefOr, ObjectBuilder},
-    Modify, OpenApi,
+    Modify, OpenApi, ToSchema,
 };
 
 use self::api_resource::RoutedResource;
 use crate::{
     ontology::domain_validator::DomainValidator,
     store::{
         crud::Read,
-        query::{Expression, ExpressionError, ResolveError},
+        query::{Expression, ExpressionError, Filter, QueryRecord, ResolveError},
         StorePool,
     },
+    subgraph::{GraphResolveDepths, NewStructuralQuery},
 };
 
+#[derive(Debug, Deserialize, ToSchema)]
+#[serde(
+    deny_unknown_fields,
+    rename_all = "camelCase",
+    rename = "StructuralQuery"
+)]
+pub struct StructuralQuery {
```

Comment:
Is this old SQ struct moved here because it isn't used anywhere else and is gone in next PR?

Reply:
Exactly, this is only used in the OpenAPI spec.